app/tests/routers/test_users.py

```diff
@@ -63,12 +63,14 @@ async def test_update_user_profile_display_name(
     async def test_update_user_profile_description(
         self, app: FastAPI, db: Database, authorized_client: AsyncClient, server: Server
     ):
-        response = await authorized_client.get(f"/users/me?server_id={str(server.id)}")
+        response = await authorized_client.get("/users/me")
```

Comment:
I had to also remove the `server_id` query parameter, because the `description` field is only available for User updates, and having the `server_id` param would've triggered a ServerMember update.